networks/mainnet-abi.json

```diff
@@ -1,4 +1,370 @@
 {
+    "GovernorBravo": [
```

Comment:
Actually looking at this, `GovernorBravo` doesn't have the propose function in this abi. How did we generate it?

Reply:
ah, was missing delegate functions